modules/programs/neovim.nix

```diff
@@ -213,11 +213,17 @@ in
       extraLuaConfig = mkOption {
         type = types.lines;
         default = "";
-        example = ''
-          vim.opt.nobackup = true
+        example = lib.literalExpression ''
+          let
+                nvimEarlyInit = lib.mkOrder 500 "set rtp+=vim.opt.rtp:prepend('/home/user/myplugin')";
+                nvimLateInit = lib.mkAfter 1000 "vim.opt.signcolumn = 'auto:1-3'";
```

Comment:
`mkAfter` is a single-argument function.

I assume you wanted to either use `mkOrder 1000 "..."` (but 1000 is the default priority) or `mkAfter "..."`.